app/serializers/topic_list_item_serializer.rb

```diff
@@ -72,4 +75,12 @@ def tags
     object.tags.map(&:name)
   end
 
+  def include_featured_link?
+    SiteSetting.topic_featured_link_enabled
+  end
+
+  def featured_link
+    object.custom_fields[DiscourseFeaturedLink::CUSTOM_FIELD_NAME]
```

Comment:
Lets extract this `.custom_fields[DiscourseFeaturedLink::CUSTOM_FIELD_NAME]` into a method under the model